razorpay-quick-payments.php

```diff
@@ -275,15 +275,17 @@ function wpCheckRazorpayResponse()
 
         protected function getPostAttributes()
         {
-            $attributes = array(
-                'razorpay_payment_id' => sanitize_text_field($_POST['razorpay_payment_id']),
-                'razorpay_order_id'   => $_SESSION['razorpay_order_id'],
-                'razorpay_signature'  => sanitize_text_field($_POST['razorpay_signature'])
-            );
+            if (isset($_POST['rzp_QP_form_submit']))
```

Comment:
@captn3m0 Strict check?

Reply:
isset is gauranteed to return a boolean so I think it is fine.